dags/hivemind_etl_helpers/src/db/telegram/utils/module.py

```
@@ -0,0 +1,119 @@
+        check if the platform exist in a module holding the community id
+        """
+        document = self._client[self.database][self.collection].find_one(
+            {
+                "community": ObjectId(self.community_id),
+                "options.platforms.platform": ObjectId(self.platform_id)
+            },
+            {
+                "_id": 1,
+            },
+        )
+        return True if document else False
```

Ruff

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)